src/ngAria/aria.js

```diff
@@ -83,7 +83,8 @@ function $AriaProvider() {
     ariaMultiline: true,
     ariaValue: true,
     tabindex: true,
-    bindKeypress: true
+    bindKeypress: true,
+    keypressNodeNames: ['DIV', 'LI']
```

Comment:
I still kind of feel like this is a really weird API. Chances are, you're going to want emulated clicks for any element that doesn't produce them on their own, right? if this array gets out there and people start using it, we won't be able to get rid of it later. Lets make sure this is needed before doing it